package.json

```diff
@@ -38,6 +38,7 @@
         "CI/CD"
     ],
     "activationEvents": [
+        "onLanguage:yaml",
```

Comment:
Why the new activation event for YAML files?

Reply:
When I open pipelines from our repos in VS Code, the language is yaml. The extension was not getting activated for those pipeline files. Hence, I had to introduce yaml

Reply:
I do think we should keep this as Azure Pipelines-only. If there's specific standards for what 1ES PT files should be named, then that can be added to the filename patterns: https://github.com/microsoft/azure-pipelines-vscode/blob/main/package.json#L58

Reply:
We do not follow a specific standard while naming 1ESPT files. That's why I had added the language as yaml and not in the file pattern (as we will need to add '*.yml' )
@flyingUnderTheRadar Please correct me if I am wrong

Reply:
Nothing to add here since there really is no set pattern or name for the 1ES PT based pipelines. Is the concern here that we will activate for every `yaml` file?

Reply:
> Is the concern here that we will activate for every yaml file?

Indeed - you're right that the experience today is a manual process, but VS Code is pretty good at remembering your choice so you don't need to do it every time. If we try to activate on every YAML file, that means we'll also try to load the schema for non-Azure Pipelines files (GitHub Actions, for example). Which decidedly isn't what we want:
![gh-actions-with-az-pipelines-schema](https://github.com/microsoft/azure-pipelines-vscode/assets/2766036/32dc507a-3d7d-4f49-aef0-71103761b2b7)

Reply:
makes sense. Removed the code change :)